### PR TITLE
Initial new appointment form page for VAOS

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
Part of the VAOS application is a multi-page new appointment form. The designs we have so far don't really fit the existing form application designs, plus I don't believe we'll want to use SiP (much of the info is time sensitive and would go stale quickly). So, to start, we're going to use `SchemaForm` and build the routing and page flow around that.

This may change over time, but thought this was a good starting point.

## Testing done
Local testing, some unit tests

## Screenshots
Screen Shot 2019-08-14 at 3 59 55 PM

## Acceptance criteria
- [x] There's a form page

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/routes.jsx [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 import Index from './containers/Main';
+import NewAppointmentLayout from './components/NewAppointmentLayout';
+import TypeOfAppointmentPage from './containers/TypeOfAppointmentPage';
 
-const routes = {
-  component: VAOSApp,
-  childRoutes: [{ path: '/', component: Index }],
-};
+const routes = (
+  <Route path="/" component={VAOSApp}>
```
Comment (jbalboni) [inline src/applications/vaos/routes.jsx]: This format seems to work better when you have index routes.

**Review by narin — APPROVED** [review]
LGTM

File: src/applications/vaos/routes.jsx (hunk 1 above)
Comment (cvalarida) [reply src/applications/vaos/routes.jsx]: Did you use the yeoman generator to spin up the app initially?

**Review by cvalarida — APPROVED** [review]
Interesting! Making a form without the forms library, but still using Schemaform. I'm really interested to see how this works for you. (And if it's worth us making a forms-lite library for just this use case.)

File: src/applications/vaos/routes.jsx (hunk 1 above)
Comment (jbalboni) [reply src/applications/vaos/routes.jsx]: No, @narin didn't know about it and create the app from scratch. I did run it recently when I was doing some exploratory work and found a couple things that I need to put into tickets for you all, though.

**Pull request merged**